.eslintignore

```diff
@@ -5,3 +5,5 @@ coverage/
 .eslintrc.js
 jest.config.js
 babel.config.js
+
+examples/webpack.config.js
```

Comment:
They were node issues. `require is not defined`, mostly.

Rather than add `@types/node` for this one file, I decided it was safe enough to just not lint it. :man_shrugging: 

I'm not opposed to adding `@types/node`, but it's just one more dependency.

Reply:
If adding `@types/node` to `devDependencies` will make the linting errors go away then I'd prefer that, but this isn't the hill I'm going to die on.